src/main/java/de/tum/in/www1/artemis/domain/TextCluster.java

```diff
@@ -103,6 +103,10 @@ public TextCluster blocks(List<TextBlock> textBlocks) {
         return this;
     }
 
+    /**Adds a TextBlock to the cluster and returns itself afterwards
+     * @param textBlock Textblock to be added
+     * @return the cluster
+     */
```

Comment:
Can we fix the style of this comment? (Same in lines 148-152)
I would prefer this:
```java
    /**
     * Adds a TextBlock to the cluster and returns itself afterwards
     * @param textBlock Textblock to be added
     * @return the cluster
     */
```

Reply:
Done